src/server/qgsrequesthandler.cpp

```diff
@@ -209,9 +209,10 @@ void QgsRequestHandler::parseInput()
       int column = -1;
       if ( !doc.setContent( inputString, true, &errorMsg, &line, &column ) )
       {
-        // XXX Output error but continue processing request ?
+        // Output Warning about POST without XML content
         QgsMessageLog::logMessage( QStringLiteral( "Warning: error parsing post data as XML: at line %1, column %2: %3. Assuming urlencoded query string sent in the post body." )
```

Comment:
```suggestion
        QgsMessageLog::logMessage( QStringLiteral( "Error parsing post data as XML: at line %1, column %2: %3. Assuming urlencoded query string sent in the post body." )
```
And translatable string ?

Reply:
@nicogodet I don't think any QGIS Server string is translatable. It will be for a next PR.

@elpaso what do you think about it ?

Reply:
@rldhont You are right, my bad.

Reply:
@nicogodet no worries, it is an enhancement to keep in mind.

I have integrated your suggestion.

Reply:
@rldhont  Looks good to me